__test__/creds/VerifiableCredential.test.js

```diff
@@ -811,6 +811,104 @@ describe('Unit tests for Verifiable Credentials', () => {
     done();
   });
 
+  it('should check all verifications with success', async (done) => {
+    const credentialContents = fs.readFileSync('__test__/creds/fixtures/VCPermanentAnchor.json', 'utf8');
+    const credentialJson = JSON.parse(credentialContents);
+    const cred = VC.fromJSON(credentialJson);
+
+    const subject = signAttestationSubject(cred.proof.anchor.subject, XPVT1, XPUB1);
+    const signedCred = VC.fromJSON(_.merge({}, cred, { proof: { anchor: { subject } } }));
+
+    expect(signedCred).toBeDefined();
+    expect(signedCred.proof.anchor).toBeDefined();
+
+    const requestorId = 'ANY_REQUESTOR_ID_12345';
+    const requestId = new Date().getTime(); // simulate an nonce ID
+    signedCred.grantUsageFor(requestorId, requestId, { pvtKey: XPVT1 });
+
+    expect(signedCred.granted).not.toBeNull();
+
+    const validation = await signedCred.verifyAll({ requestorId, requestId });
+    expect(validation).toBeTruthy();
+    done();
+  });
+
+  it('should check all verifications and fail - tampered grant', async (done) => {
+    const credentialContents = fs.readFileSync('__test__/creds/fixtures/VCPermanentAnchor.json', 'utf8');
```

Comment:
I think that a require in __test__/creds/fixtures/VCPermanentAnchor.json is better.

Reply:
The anchor on this test is not checked. it fails before that.